### DISCUSSION TITLE
【SOUNDRAW】AI 音樂生成器

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS3073-C/Python-Community-GS3073-C#27] [科技新知]
## 簡介
SOUNDRAW 是一個 AI 音樂生成器，允許所有創作者自由定制獨特、免版稅音樂的長度、組成和樂器
實際聽了DEMO音樂後，覺得還可以，因為沒有讓我感到很精豔
但是，用AI來做音樂生成，並且製作成web app來提供商品服務，這件事情很棒。
如果將來同學們的專題是做出AI商業服務網站，這種作品集對將來就業或升學一定很有說服力阿~

## 參考網站：
1. 官網：https://soundraw.io/
2. 某一個販售網站：https://appsumo.com/products/soundraw/

**Comment by awehao** [comment]
好新奇，不但可以選擇自己想要的曲風，還可以選擇樂器，雖然感覺裡面的音樂有些制式化，但也是一個很大的突破

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by boson13579** [comment]
用AI寫出來的音樂可以模仿特定音樂家的風格，也可以創造出融合了不同風格的音樂，這是真人演奏家無法達到的境界，因為音樂家會被侷限在既有的圈裡難以做出突破，或說突破了但效果甚微，AI則完全沒有這個問題。

**Comment by yjpxi** [comment]
像我們學校高中畢業歌原本打算用AI作曲，歌詞 旋律都是ai來幫你完成，但是聽完AI唱的版本時還是覺得沒有人唱的那麼有溫度，就放棄AI作曲了。雖然AI會生成給你不同條件的歌曲，但是他生成的音樂秒數只有短短的90秒，根本不足以做成一首歌。另外我覺得即使SOUNDRAW生成的音樂是免版稅的，但使用 AI 技術可能會涉及到訓練數據的版權問題，使用 AI 來創作藝術作品可能會引發一些倫理問題，會對人類音樂家造成職業的影響。
